tools/generator/autorest_ext/minorVersion.go

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+package autorest_ext
```

Comment:
This function should be moved to another package since this actually is not something related with `autorest`

Reply:
move to generator/utils since it is a tooling function